commit 15d50bcbfb6648a1ed846b6ca4e15af3b8bd34f9
Author: Isaac Davis <isaac.davis@joyent.com>
Date:   2019-05-03T17:37:16+00:00 (5 months ago)
    
    TOOLS-2251 Makefile.go_prebuilt.targ references local copy of download_go instead of eng.git path
    Reviewed by: Tim Foster <tim.foster@joyent.com>
    Approved by: Tim Foster <tim.foster@joyent.com>

diff --git a/tools/mk/Makefile.go_prebuilt.targ b/tools/mk/Makefile.go_prebuilt.targ
index 43c0757..0632da6 100644
--- a/tools/mk/Makefile.go_prebuilt.targ
+++ b/tools/mk/Makefile.go_prebuilt.targ
@@ -51,7 +51,7 @@ endif
 $(GO_TARBALL):
 	rm -f $@
 	mkdir -p $(@D)
-	tools/download_go $(GO_PREBUILT_VERSION) solaris amd64 $(CACHE_DIR)
+	./deps/eng/tools/download_go $(GO_PREBUILT_VERSION) solaris amd64 $(CACHE_DIR)
 
 #
 # Extract the Go toolchain.  This stamp includes the version number of the
